week1/react-exercises/src/1-hobbies/Hobbies.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+const HobbyList = () => {
+  const hobbies = [
+    "Surfing",
+    "Rock climbing",
+    "Mountain biking",
+    "Breakdancing",
+  ];
+
+  return (
+    <div className="hobbies">
+      <h1>Extreme Hobbies:</h1>
+      <ul>
+        {hobbies.map((hobby, i) => (
+          <Hobbies key={i} hobby_={hobby} />
```

Comment:
- I would call this component `<Hobby/>` because you are displaying one hobby. 
- When there is no `id` use another unique key, don't use the `index`
-  `hobby_` is a typo? 

```suggestion
          <Hobby key={hobby} hobby={hobby} />
```